src/AccessibilityInsights.SharedUx/FileBug/BugReporter.cs

```diff
@@ -14,125 +13,41 @@ namespace AccessibilityInsights.SharedUx.FileBug
     /// </summary>
     static public class BugReporter
     {
-        private static IBugReporting BugReporting => Container.GetDefaultInstance().BugReporting;
+        public static IIssueReporting IssueReporting { get; set; }
 
-        public static bool IsEnabled => BugReporting != null;
+        public static bool IsEnabled => (IssueReporterManager.GetInstance().GetIssueFilingOptionsDict() != null && IssueReporterManager.GetInstance().GetIssueFilingOptionsDict().Any());
 
-        public static bool IsConnected => IsEnabled && BugReporting.IsConnected;
+        public static bool IsConnected => IsEnabled && (IssueReporting == null ? false : IssueReporting.IsConfigured);
 
 #pragma warning disable CA1819 // Properties should not return arrays
-        public static byte[] Avatar => IsEnabled ? BugReporting.Avatar?.ToArray() : null;
+        public static byte[] Logo => IsEnabled ? IssueReporting.Logo?.ToArray() : null;
 #pragma warning restore CA1819 // Properties should not return arrays
 
-        public static string DisplayName => IsEnabled ? BugReporting.DisplayName : null;
+        public static string DisplayName => IsEnabled ? IssueReporting.ServiceName : null;
 
-        public static string Email => IsEnabled ? BugReporting.Email : null;
-
-        public static Task ConnectAsync(Uri uri, bool prompt)
+        public static Dictionary<Guid, IIssueReporting> GetIssueReporters()
         {
-            if (IsEnabled)
-                return BugReporting.ConnectAsync(uri, prompt);
-
-            return Task.CompletedTask;
+            return IssueReporterManager.GetInstance().GetIssueFilingOptionsDict();
         }
 
-        public static void FlushToken(Uri uri)
+        public static Task RestoreConfigurationAsync(string serializedConfig)
         {
-            if (IsEnabled)
-                BugReporting.FlushToken(uri);
-        }
-
-        public static Task PopulateUserProfileAsync()
-        {
-            if (IsEnabled)
-                return BugReporting.PopulateUserProfileAsync();
-
+            if (IsEnabled && IssueReporterManager.SelectedIssueReporterGuid != null)
+                return IssueReporting.RestoreConfigurationAsync(serializedConfig);
             return Task.CompletedTask;
         }
 
-        public static void Disconnect()
+        public static IIssueResult FileIssueAsync(IssueInformation issueInformation)
```

Comment:
If we intend to return the blocked result then we should remove `async` from the method name to avoid confusion

Reply:
Noted. Will fix one the integration is finished.